stl/inc/algorithm

```diff
@@ -4805,7 +4810,7 @@ _EXPORT_STD template <class _ExPo, class _BidIt, class _FwdIt, _Enable_if_execut
 _FwdIt reverse_copy(_ExPo&&, _BidIt _First, _BidIt _Last, _FwdIt _Dest) noexcept /* terminates */ {
     // copy reversing elements in [_First, _Last)
     // not parallelized as benchmarks show it isn't worth it
-    _REQUIRE_PARALLEL_ITERATOR(_BidIt);
+    static_assert(_Is_ranges_bidi_iter_v<_BidIt>, "This algorithm requires bidirectional iterators or stronger.");

```

Comment:
I find it mildly disturbing that this isn't a named-requirement-macro like the other cases. I have a mild preference to use a macro for consistency even if this is the only use.

Reply:
I've made it so.